### PR TITLE
Add support for running test sites on IIS from VS

### PR DESCRIPTION
**Pull request by pakrym**
Supports both x64 and x86

### CODE REVIEW

File: build/testsite.props [hunk 1]
```diff
@@ -4,26 +4,37 @@
...
+    <None Include="$(MSBuildThisFileDirectory)..\src\RequestHandler\bin\$(Configuration)\$(NativeFolder)\aspnetcorerh.dll" CopyToOutputDirectory="PreserveNewest" Visible="true" Link="$(NativePlatform)\%(FileName)%(Extension)" />
+    <None Include="$(MSBuildThisFileDirectory)..\src\RequestHandler\bin\$(Configuration)\$(NativeFolder)\aspnetcorerh.pdb" CopyToOutputDirectory="PreserveNewest" Visible="true" Link="$(NativePlatform)\%(FileName)%(Extension)" />
+    <None Include="$(MSBuildThisFileDirectory)..\src\AspNetCore\bin\$(Configuration)\$(NativeFolder)\aspnetcore.dll" CopyToOutputDirectory="PreserveNewest" Visible="true" Link="$(NativePlatform)\%(FileName)%(Extension)" />
+    <None Include="$(MSBuildThisFileDirectory)..\src\AspNetCore\bin\$(Configuration)\$(NativeFolder)\aspnetcore.pdb" CopyToOutputDirectory="PreserveNewest" Visible="true" Link="$(NativePlatform)\%(FileName)%(Extension)" />
+  </ItemGroup>
+
   <PropertyGroup Condition="'$(NativePlatform)' == 'x86'">
-    <IISExpress>$(MSBuildProgramFiles32)\IIS Express\iisexpress.exe</IISExpress>
+    <IISExpressPath>$(MSBuildProgramFiles32)\IIS Express\iisexpress.exe</IISExpressPath>
```
Comment (jkotalik) [inline build/testsite.props]: Nit: On a 32 bit machine, what does this return? I think it returns Program Files.

File: build/testsite.props [hunk 2]
```diff
@@ -4,26 +4,37 @@
...
+    <IISPath>$(SystemRoot)\SysWOW64\inetsrv\w3wp.exe</IISPath>
```
Comment (jkotalik) [inline build/testsite.props]: Same here, I don't know what SysWoW64 is on a 32 bit machine.

File: build/testsite.props (hunk 1 above)
Comment (pakrym) [reply build/testsite.props]: Yes, which would be correct.

File: build/testsite.props (hunk 2 above)
Comment (pakrym) [reply build/testsite.props]: I would like to see person trying to open this solution on 32 bit windows :)

Comment (jkotalik) [reply build/testsite.props]: Yeah its minor, I don't care much.

**Pull request merged**